pavelib/utils/test/utils.py

```diff
@@ -150,3 +157,35 @@ def check_firefox_version():
                 debian_path=debian_path
             )
         )
+
+
+@task
+@cmdopts([
+    ("compare-branch=", "b", "Branch to compare against, defaults to origin/master"),
+], share_with=['coverage'])
+@timed
+def fetch_coverage_test_selection_data(options):
+    """
+    Set up the datafiles needed to run coverage-driven test selection (who-tests-what)
+    """
+
+    try:
+        os.makedirs(COVERAGE_CACHE_BASEPATH)
+    except OSError:
+        pass  # Directory already exists
+
+    sh('git diff $(git merge-base {} HEAD) > {}/{}'.format(
+        getattr(options, 'compare_branch', 'origin/master'),
+        COVERAGE_CACHE_BASEPATH,
+        WHO_TESTS_WHAT_DIFF
+    ))
+    get_file_from_s3(COVERAGE_CACHE_BUCKET, COVERAGE_CACHE_BASELINE, COVERAGE_CACHE_BASEPATH)
+
+
+@task
+def upload_coverage_to_s3(options):
+    upload_to_s3(
+        COVERAGE_CACHE_BASELINE,
+        '{}/{}'.format(COVERAGE_CACHE_BASEPATH, 'reports/.coverage'),
```

Comment:
The data file is named differently now, isn't it? Can we wild-card this?

Reply:
Fixed?

Reply:
I admit to being confused about when it is called `.coverage.lms` and when it is called `lms.coverage`.  But this matches the .coveragerc files now.